packages/serialize/src/index.js

```diff
@@ -357,7 +357,10 @@ export const serializeStyles = function(
 
   let match
   // https://esbench.com/bench/5b809c2cf2949800a0f61fb5
-  while ((match = labelPattern.exec(styles)) !== null) {
+  // styles.replace(/;$/, '').concat(';') safe add semicolon to fix #2311
+  while (
+    (match = labelPattern.exec(styles.replace(/;$/, '').concat(';'))) !== null
```

Comment:
Could you try an alternative fix?
```diff
-let labelPattern = /label:\s*([^\s;\n{]+)\s*;/g
+let labelPattern = /label:\s*([^\s;\n{]+)\s*(;|$)/g
```

Reply:
Of course, thanks for code review, when I try to solve this issue, I found many ways to fix, so I am confused which one is  correct and suitable solution.

Reply:
@Andarist Hi, I try the alternative fix, it works well. And I have a question, Is CI broken ?
netlify's documentaion website auto deploy looks like not ok ...
```bash
1:03:48 AM: 🎁 info building bundles!
1:03:57 AM: 🎁 success built bundles!
1:03:58 AM: > emotion-monorepo@10.0.0 build:site /opt/build/repo
1:03:58 AM: > cd site && yarn build:site
1:03:58 AM: yarn run v1.7.0
1:03:58 AM: $ gatsby build
1:04:02 AM: success open and validate gatsby-configs - 0.316s
1:04:03 AM: error
1:04:03 AM: Something went wrong installing the "sharp" module
1:04:03 AM: Cannot find module '../build/Release/sharp.node'
1:04:03 AM: Require stack:
1:04:03 AM: - /opt/build/repo/node_modules/sharp/lib/constructor.js
```
Maybe need to fix emotion-js infra CI/CD ?

Reply:
Yes, something is off with Netlify's setup. I didn't have time to dig into this.